Definitions.md

```diff
@@ -4,7 +4,7 @@
 | ------------- | ------------- | ----- | ----- |
 | event/run/lumi | - | - | general event info |
 | vtx | `g4SimHits` | `std::vector<SimVertex>` | primary vertex position |
-| genpart | `g4SimHits` | `std::vector<SimTrack>` | truth level tracks/particles |
+| genpart | `g4SimHits` | `std::vector<SimTrack>` | truth level tracks/particles and information related to their extrapolation towards HGCAL. In particular reachedEE==2 indicates that the particles reached HGCAL while reachedEE==1 is for barrel and reachedEE==0 is for the other cases |
```

Comment:
Could you please use `reachedEE==1` instead of unformatted "reachedEE==1" etc. and also write "barrel calorimeter" instead of "barrel" only for better clarity?